deploy/13-verification.ts

```diff
@@ -0,0 +1,34 @@
+// @ts-nocheck
+import  "@nomiclabs/hardhat-ethers";
+import hre from 'hardhat';
+import fs from 'fs';
+import path from 'path';
+
+async function main(){
```

Comment:
I see that all the deployment scripts (in the `/deploy` folder) have this snippet commented:

``` 
// try {
  //   await new Promise((r) => setTimeout(r, 30000));
  //   await run("verify:verify", {
  //     address: deployResult.address,
  //     constructorArguments: [clearingHouse.address],
  //   });
  // } catch (error) {
  //   console.log(error);
  // }
```
  
What do you think about extracting it to the `/scripts` folder and make it reusable?

For example something like this:


```
import { run } from 'hardhat';

export const verify = async (contractAddress: string, args: Array<any>): Promise<void> => {
  console.log('Verifying contract...');
  try {
    await run('verify:verify', {
      address: contractAddress,
      constructorArguments: args,
    });
     console.log("Verified done!")
  } catch (error: any) {
    if (error.message.toLowerCase().includes('already verified')) {
      console.log('Already verified!');
    } else {
      console.error(`Error occurred while verifying contract ${contractAddress}: `, error)
    }
  }
};
```

Reply:
Or maybe I misunderstood and the point of this PR is to verify several contracts into different networks? 😅

Reply:
done